libsolidity/parsing/Parser.cpp

```diff
@@ -382,6 +382,34 @@ Declaration::Visibility Parser::parseVisibilitySpecifier()
 	return visibility;
 }
 
+ASTPointer<OverrideSpecifier> Parser::parseOverrideSpecifier()
+{
+	solAssert(m_scanner->currentToken() == Token::Override, "");
+	ASTNodeFactory nodeFactory(*this);
+
+	m_scanner->next();
+
+	std::vector<ASTPointer<UserDefinedTypeName>> overrides;
+
+	if (m_scanner->currentToken() == Token::LParen)
+	{
+		m_scanner->next();
+		while (true)
+		{
+			overrides.push_back(parseUserDefinedTypeName());
+
+			if (m_scanner->currentToken() == Token::RParen)
+				break;
+
+			expectToken(Token::Comma);
+		}
+
+		expectToken(Token::RParen);
+	}
+
+	return nodeFactory.createNode<OverrideSpecifier>(move(overrides));
```

Comment:
Please check that the sourcelocation end pointer is set properly here (in both cases)

Reply:
Check as in, with a test or simply manually checking?

Reply:
looking over the code again, I think they are fine. The c'tor sets the start position, the createNode call the end position

Reply:
Ok!